### PR TITLE
feat(docs): updated order summary style guidance and replaced example…

### PR DESCRIPTION
**Pull request by adrianamorales**
**Order summary updates:**
- added typography, button and more detailed specs 
- removed an example image

**Cards:**
- added the card sketch kit
- updated example images

### CODE REVIEW

File: src/pages/experimental/order-summary-template/style.mdx [hunk 1]
```diff
@@ -1,10 +1,54 @@
 ---
-title: Order summary template
-tabs: ['Usage', 'Style']
+title: Order summary
+tabs: ['Usage', 'Style', 'Code']
```
Comment (connor-leech) [inline src/pages/experimental/order-summary-template/style.mdx]:
```suggestion
tabs: ['Usage', 'Style', 'Code']
```

This is the only page with the Code page linked as a tab. Is that intentional?

File: src/pages/experimental/order-summary-template/style.mdx [hunk 2]
```diff
@@ -1,10 +1,54 @@
...
 
-## Design spec
+## Specs
 
-![Design spec for the order summary template](/images/specs_ordersummary.png)
+<Row>
+  <Column colMd={12} colLg={12} GutterSm>
 
-<Caption>Design spec for the order summary template</Caption>
+![Order Summary Redlines](images/specs_ordersummary.png)
```
Comment (connor-leech) [inline src/pages/experimental/order-summary-template/style.mdx]:
```suggestion
![Order summary redlines](images/specs_ordersummary.png)
```

File: src/pages/experimental/order-summary-template/style.mdx [hunk 3]
```diff
@@ -1,10 +1,54 @@
...
+
+<Caption>
+  Structure and spacing measurements for small side panels | px | rem.
```
Comment (connor-leech) [inline src/pages/experimental/order-summary-template/style.mdx]:
```suggestion
  Structure and spacing measurements for small side panels | px | rem
```

File: src/pages/experimental/order-summary-template/style.mdx [hunk 4]
```diff
@@ -1,10 +1,54 @@
...
+
+## Padding
+
+Padding should be `2rem (32px)` for the top, left, right, and bottom of the content area.
+
+Buttons should have `1 rem (16px)` of padding on the left.
+
+## Color
+
+The order summary uses a gray 100 theme.
```
Comment (connor-leech) [inline src/pages/experimental/order-summary-template/style.mdx]:
```suggestion
The order summary uses a Gray 100 theme.
```

File: src/pages/experimental/order-summary-template/style.mdx [hunk 5]
```diff
@@ -1,10 +1,54 @@
...
+Background color `#171717`.
```
Comment (connor-leech) [inline src/pages/experimental/order-summary-template/style.mdx]:
```suggestion
The background color is `#171717`.
```

File: src/pages/experimental/order-summary-template/style.mdx [hunk 6]
```diff
@@ -1,10 +1,54 @@
...
+
+## Buttons
+
+The primary and secondary button within the order summary footer can be active or disabled depending on the situation. Buttons should always lie above the fold.
+Follow Carbon's guidance on button usage.
```
Comment (connor-leech) [inline src/pages/experimental/order-summary-template/style.mdx]:
```suggestion
```

File: src/pages/experimental/order-summary-template/style.mdx (hunk 1 above)
Comment (adrianamorales) [reply src/pages/experimental/order-summary-template/style.mdx]: That was accidental. I copied the markdown from the cloud pal page and forgot to remove the code tab.

File: src/pages/experimental/order-summary-template/style.mdx [hunk 7]
```diff
@@ -1,10 +1,53 @@
 ---
-title: Order summary template
-tabs: ['Usage', 'Style']
+title: Order summary
+tabs: ['Usage', 'Style', 'Code']
```
Comment (connor-leech) [inline src/pages/experimental/order-summary-template/style.mdx]:
```suggestion
tabs: ['Usage', 'Style']
```

File: src/pages/experimental/order-summary-template/style.mdx (hunk 1 above)
Comment (connor-leech) [reply src/pages/experimental/order-summary-template/style.mdx]: Ah, ok. I just proposed the change to remove the code tab from this page

File: src/pages/experimental/cards/usage.mdx [hunk 8]
```diff
@@ -151,10 +160,12 @@ Cards can be used to block groups of related content even if it doesn't fit into
...
+### Cards sketch kit
 
-- Austin Edwards
-- Kathryn Gallagher
-- Alexander Meyer
-- Ceci Moralles
-- JP Pollard
+<Row className="resource-card-group">
+  <Column colLg={4} colMd={4} noGutterSm>
+    <ResourceCard subTitle="Cards sketch kit" href="carbon-cards.sketch">
```
Comment (alisonjoseph) [inline src/pages/experimental/cards/usage.mdx]: This file doesn't seem to exist?

**Comment by connor-leech** [comment]
@adrianamorales Just checking in on that Sketch file for this PR

**Comment by alisonjoseph** [comment]
Hi @adrianamorales, any updates on this?

**Comment by alisonjoseph** [comment]
Hi @adrianamorales as there as been no activity on this PR in over a month I'm going to close, please re-open if you're able to fix the sketch link and we can get this merged in. Thanks!

**Pull request closed without merging**